FlaskTemplate/{{cookiecutter.project}}/{{cookiecutter.project_name}}/__init__.py

```diff
@@ -16,17 +15,25 @@
 
 
 def create_app(config_name):
+    """
+    Handle app initialization and create an app instance
+    """
     # config
     app = flask.Flask(__name__)
     app.config.from_object(config.config[config_name])
     config.config[config_name].init_app(app)
+
     # extensions
     db.init_app(app)
     login_manager.init_app(app)
     hashing.init_app(app)
     mail.init_app(app)
+
     # blueprints
+    from . import blueprints
```

Comment:
It seemed like the import blueprints statements that used to be at the top had issues because the blueprints relied on things like db but those hadn't been defined yet